packages/suite/src/hooks/wallet/coinmarket/form/useCoinmarketVerifyAccount.tsx

```diff
@@ -81,6 +81,8 @@ const getSuiteReceiveAccounts = ({
                 a =>
                     a.deviceState === device?.state &&
                     a.symbol === receiveNetwork &&
+                    ((isDebugOnlyAccountType(a.accountType, a.symbol) && isDebug) ||
+                        !isDebugOnlyAccountType(a.accountType, a.symbol)) &&
                     (!a.empty || a.visible || (a.accountType === 'normal' && a.index === 0)),
```

Comment:
nit: this seems like a very complex condition. Couldn't we split it to multiple constants?

Reply:
@enjojoy 🙏